synapse/tests/test_lib_cli.py

```diff
@@ -57,21 +57,6 @@ def test_cli_opts_flag(self):
             self.eq( opts.get('bar'), True )
             self.eq( opts.get('haha'), 'hoho')
 
-    def test_cli_opts_flag(self):
```

Comment:
this duplicates https://github.com/vertexproject/synapse/blob/master/synapse/tests/test_lib_cli.py#L45